incubator/barbarian/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: Barbarian is a big data system founded on Apache Hadoop offering Hive, LLAP and Tez for fast SQL datawarehousing at scale.
+name: barbarian
+version: 0.1.2-badh0.1.0
```

Comment:
Not 100% sure, but seems like this doesn't follow semantic versioning guidelines (which is a requirement).